docs/whitepapers/kalmanfilter/kalmanfilter.tex

```diff
@@ -375,51 +375,51 @@ \section{Altitude Dynamics}
     \section{Kalman filter equations}
 The Kalman filter equations for the above system are:
 \begin{align}
-    \text{Measurement} &
-    \left[
-    \begin{array}{l}
-      \hat{x}_{t{}\mid{}t}
-      {}={}
-      \hat{x}_{t{}\mid{}t-1}
-      {}+{}
-      \Sigma_{t{}\mid{}t-1}C^\intercal
-      (C\Sigma_{t{}\mid{}t-1}C^\intercal + R)^{-1}(y_t - C\hat{x}_{t{}\mid{}t-1})
-      \\
-      \Sigma_{t{}\mid{}t}
-      {}={}
-      \Sigma_{t{}\mid{}t-1}
-      {}-{}
-      \Sigma_{t{}\mid{}t-1}C^\intercal
-      (C\Sigma_{t{}\mid{}t-1}C^\intercal + R)^{-1}
-      C\Sigma_{t{}\mid{}t-1}
-    \end{array}
-    \right.
+  \text{Measurement} &
+  \left[
+  \begin{array}{l}
+    \hat{x}_{t{}\mid{}t}
+    {}={}
+    \hat{x}_{t{}\mid{}t-1}
+    {}+{}
+    \Sigma_{t{}\mid{}t-1}C^\intercal T
```

Comment:
@Yuanbwcx, Why have you added a `T` after all the `\intercal`?
`\intercal` is used for matrix/vector transpose symbol.

Reply:
> @Yuanbwcx, Why have you added a `T` after all the `\intercal`? `\intercal` is used for matrix/vector transpose symbol in LaTeX.

I follow by the file called "h14_kalman_filter" which is uploaded in teams, I thought someone forgot to put the T on.

Reply:
Did you build the LaTeX project and check the PDF?

This is what it looks like now


![Screenshot 2024-04-04 203419](https://github.com/QUB-ASL/bzzz/assets/95449123/895f3e8f-cc32-4b1b-96dd-f90db8025ef2)

The yellow Highlight is `\intercal` the pink is `T`

Reply:
> Did you build the LaTeX project and check the PDF?
> 
> This is what it looks like now
> 
> ![Screenshot 2024-04-04 203419](https://private-user-images.githubusercontent.com/95449123/319749839-895f3e8f-cc32-4b1b-96dd-f90db8025ef2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIyNjk2NjUsIm5iZiI6MTcxMjI2OTM2NSwicGF0aCI6Ii85NTQ0OTEyMy8zMTk3NDk4MzktODk1ZjNlOGYtY2MzMi00YjFiLTk2ZGQtZjkwZGI4MDI1ZWYyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA0VDIyMjI0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxYzVlMDczYTBkMWM0ZTIwNDA1YWU3NTg3OTMyYTA1MDFjYWIxZjRmNDM4MWY0ZjY5Y2I5ZDRhNDJkNzk5MjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kZo-086tyFBs6PPX136x1AVy9AJpkvofudU17YkqJfI)
> 
> The yellow Highlight is `\intercal` the pink is `T`

Sorry my bad, corrected it now.